modules/gmc-agent/src/main/java/org/gridgain/dto/span/SpanBatch.java

```diff
@@ -27,9 +27,9 @@
 import java.util.List;
 
 /**
- * DTO for span list.
+ * DTO for batch of span list.
  */
-public class SpanList implements Externalizable {
+public class SpanBatch implements Externalizable {
```

Comment:
I don't see any reason of using this wrapper over List. Let's remove it

Reply:
Done.